### PR TITLE
feat: add linear search algorithm in python

### PR DESCRIPTION
**Pull request by saladinreborn**
added algorithm for python

### CODE REVIEW

**Comment by Matrix278** [comment]
Is it working, can't get the result?

File: linearSearch/linearSearch.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+def linearSearch(arr, item):
+  for i in range(0, len(arr)):
+    if (arr[i] == item):
+      return i
+  return -1
+
+linearSearch([1, 2, 3, 5, 7, 11, 13, 17, 23, 31], 13)
```
Comment (Matrix278) [inline linearSearch/linearSearch.py]: Please change to this line
```suggestion
print("Element found at index "+str(linearSearch([1, 2, 3, 5, 7, 11, 13, 17, 23, 31], 13)))
```

Comment (saladinreborn) [reply linearSearch/linearSearch.py]: Ok, sure. I'll update the code.

**Comment by saladinreborn** [comment]
Hi @Matrix278 , I've updated the code by following your suggestion. Now, if you run it, the result would be printed in the terminal.

**Comment by Matrix278** [comment]
Thanks for your contribution

**Pull request merged**